web/src/components/TaxRegistrationModal.tsx

```diff
@@ -131,12 +131,12 @@ export const TaxRegistrationModal = (props: Props): ReactElement => {
             <OnboardingBusinessName onValidation={onValidation} fieldStates={fieldStates} required />
           </>
         )}
-        {showTaxIdField() && (
-          <>
-            <FieldLabelModal fieldName="taxId" />
-            <OnboardingTaxId onValidation={onValidation} fieldStates={fieldStates} required />
-          </>
-        )}
+        <>
+          <FieldLabelModal fieldName="taxId" />
+          <br />
```

Comment:
In this place (and the others as well), we should use margins to control spacing rather than inserting an empty line